apple/Brushes/RNSVGSolidColorBrush.m

```diff
@@ -13,48 +13,49 @@
 
 @implementation RNSVGSolidColorBrush
 {
-    CGColorRef _color;
+    UIColor *_color;
 }
 
 - (instancetype)initWithArray:(NSArray<RNSVGLength *> *)array
 {
     if ((self = [super initWithArray:array])) {
-        _color = CGColorRetain([RCTConvert RNSVGCGColor:array offset:1]);
+        _color = [RCTConvert RNSVGUIColor:array offset:1];
     }
     return self;
 }
 
 - (instancetype)initWithNumber:(NSNumber *)number
 {
     if ((self = [super init])) {
-        _color = CGColorRetain([RCTConvert CGColor:number]);
+        _color = [RCTConvert UIColor:number];
     }
     return self;
 }
 
 - (void)dealloc
 {
-    CGColorRelease(_color);
+    _color = nil;
 }
 
 - (CGColorRef)getColorWithOpacity:(CGFloat)opacity
 {
-    return CGColorCreateCopyWithAlpha(_color, opacity * CGColorGetAlpha(_color));
+    CGColorRef baseColor = _color.CGColor;
+    CGColorRef color = CGColorCreateCopyWithAlpha(baseColor, opacity * CGColorGetAlpha(baseColor));
+    CGColorRelease(baseColor);
```

Comment:
We should not release it since it will release `CGColor` prop of `_color`, so trying to use `_color.CGColor` again will lead to crash.

Reply:
Are you sure about that? Why would releasing _baseColor affect _color?

Reply:
Oh, now I see, good catch 👍